tests/test_generators.py

```diff
@@ -1226,7 +1226,43 @@ def test_make_honeycomb(self):
         self.assertEqual(G.number_of_edges(),(7*6+6*4+6*3)//2)
 
     def create_channel(self):
-        print('Add test')
+        # Test some defaults
+        c, cp, r = dimod.generators.mimo.create_channel()[0]
+        self.assertEqual(cp, 2)
+        self.assertEqual(c.shape, (1, 1))
+        self.assertEqual(type(r), np.random.mtrand.RandomState)
+
+        c, cp, _ = dimod.generators.mimo.create_channel(5, 5, 
+            F_distribution=("normal", "real"))
+        self.assertTrue(np.isin(c, [-1, 1]).all())
+        self.assertEqual(cp, 5)
+
+        c, cp, _ = dimod.generators.mimo.create_channel(5, 5, 
+            F_distribution=("binary", "complex"))
+        self.assertTrue(np.isin(c, [-1-1j, -1+1j, 1-1j, 1+1j]).all())
+        self.assertEqual(cp, 10)
+
+        n_trans = 40
+        c, cp, _ = dimod.generators.mimo.create_channel(30, n_trans, 
+            F_distribution=("normal", "real"))
+        self.assertLess(c.mean(), 0.2)  
+        self.assertLess(c.std(), 1.3)    
+        self.assertGreater(c.std(), 0.7)
+        self.assertEqual(cp, n_trans)
+
+        c, cp, _ = dimod.generators.mimo.create_channel(30, n_trans, 
+            F_distribution=("normal", "complex"))
+        self.assertLess(c.mean().complex, 0.2)  
+        self.assertLess(c.real.std(), 1.3)    
+        self.assertGreater(c.real.std(), 0.7)
+        self.assertEqual(cp, 2*n_trans)
+
+        c, cp, _ = dimod.generators.mimo.create_channel(5, 5, 
+            F_distribution=("normal", "real"), 
+            attenuation_matrix=np.array([[1+1j, 0.5+0.5j],[2, 3]]))
```

Comment:
@jackraymond, for this case I noticed that the channel power, [defined here](https://github.com/jackraymond/dimod/blob/f393af1ea4cb19270ce014778c3abd9217a834d2/dimod/generators/mimo.py#L254), returns a complex value. Is that desired?

Reply:
I didn't foresee anyone using non-real values for the attenuation matrix, since each component measures transmission power which can't be complex. Could be enforced with a test.

Reply:
Updated and added a check